decidim-admin/app/commands/decidim/admin/bulk_unreport_users.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class BulkUnreportUsers < Decidim::Command
+      # Public: Initializes the command.
+
```

Comment:
Need some method definition. 
Also, refer to user as current_user + needed changes 
```suggestion

      # Public: Initializes the command.
```

Reply:
Done

Reply:
Rubocop is not letting me add an indentation here.